### PR TITLE
linting: pylint refresh

### LINKED ISSUE #2781: pylint: pylint rules should be refreshed and enforced in CI

### PR DESCRIPTION
**Pull request by tmc**
## Status

Ready for review

## Description of Changes

Fixes #2781.

Changes proposed in this pull request:

Freshens and enforces pylint lint rules.

## Testing

CI

## Deployment

N/A

### CODE REVIEW

**Comment by dachary** [comment]
Quoting @tmc regarding the **store: import secure_filename properly** commit:

> the werkzeug import worked because the init.py of it does some crazy metaprogramming shit that pylint doesn't interpret

File: securedrop/Makefile [hunk 1]
```diff
@@ -1,6 +1,16 @@
 TAG ?= $(shell git rev-parse HEAD)
 IMAGE = securedrop
 
+.PHONY: lint
+lint:
+	pylint --reports=no --errors-only \
+		--disable=import-error \
+		./*.py ./**/*.py
```
Comment (dachary) [inline securedrop/Makefile]: <pre>
./**/*.py
</pre>

is the same as


<pre>
./*/*.py
</pre>

and it won't catch the *.py that are nested at the third level.

**Comment by dachary** [comment]
@tmc noticed there are some files missing

**Comment by tmc** [comment]
pushed find+rsync fix.

**Comment by dachary** [comment]
Documentation regarding Mixin & pylint at https://pylint.readthedocs.io/en/latest/faq.html#how-do-i-avoid-access-to-undefined-member-messages-in-my-mixin-classes

**Pull request merged**